bin/index.js

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env node
+var DraftLog = require('draftlog').into(console).addLineListener(process.stdin)
```

Comment:
Would you mind you adding `;` to every line and make each `if` have a block `{ }` to keep consistent code style?